db/controllers/get-topics-controller.js

```diff
@@ -0,0 +1,11 @@
+const getTopicsModel = require('../models/get-topics-model')
+
+exports.getTopicsController = (req, res, next) => {
+    getTopicsModel()
+    .then((topics) => {
+      res.status(200).send(topics)
```

Comment:
Please do send the response in an object with a key of topics. So it should look like `{topics: topics}` or `{ topics }` for short